src/applications/gi-sandbox/utils/helpers.js

```diff
@@ -214,3 +218,47 @@ export const buildSearchFilters = filters => {
 
   return searchFilters;
 };
+
+export const getPosition = () => {
+  return new Promise((resolve, reject) => {
+    const onSuccess = position => {
+      const lat = position.coords.latitude;
+      const lng = position.coords.longitude;
+      const pos = [lat, lng];
+
+      resolve(pos);
+    };
+
+    const onError = () => {
+      reject();
+    };
+
+    navigator.geolocation.getCurrentPosition(onSuccess, onError);
+  });
+};
+export const searchCriteraFromCoords = async (longitude, latitude) => {
+  const BOUNDING_RADIUS = 0.75;
+  const response = await mbxClient
+    .reverseGeocode({
+      query: [longitude, latitude],
+      types: ['address'],
+    })
+    .send();
+  // TODO: display error message if geolocation fails?
+  // .catch(error => error);
+
+  const features = response.body.features;
+  const placeName = features[0].place_name;
+  const coordinates = features[0].center;
+
+  return {
```

Comment:
why return all this info?
you already have the latitude and longitude, this should just return the lat/long's place name